main.go

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
We should not add a main.go here.  Instead, let's have travis prove it can build the sample.  That will show that we can link to the library.